From ad50d3ed2ddfe11cd07954786b96725602fb4ddd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Alexander=20F=2E=20R=C3=B8dseth?= <rodseth@gmail.com>
Date: Tue, 5 Jan 2021 12:57:27 +0100
Subject: [PATCH] Use stdbool.h istead of redefining bool

Redefining bool causes errors when used together with ie. GCC 10.2.0
---
 pretty-printer/src/PrettyPrinter.h | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/pretty-printer/src/PrettyPrinter.h b/pretty-printer/src/PrettyPrinter.h
index 268986e95..4200db612 100644
--- a/pretty-printer/src/PrettyPrinter.h
+++ b/pretty-printer/src/PrettyPrinter.h
@@ -29,6 +29,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <stdarg.h>
+#include <stdbool.h>
 
 #ifdef HAVE_GLIB
 #include <glib.h>
@@ -52,15 +53,13 @@
 #define TRUE !(FALSE)
 #endif
 
-typedef unsigned int bool;
-
 /*========================================== STRUCTURES =======================================================*/
 
 /**
  * The PrettyPrintingOptions struct allows the programmer to tell the
  * PrettyPrinter how it must format the XML output.
  */
-typedef struct 
+typedef struct
 {
       const char* newLineChars;                                                             /* char used to generate a new line (generally \r\n) */
       char indentChar;                                                                      /* char used for indentation */

